spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -371,6 +371,62 @@ describe('GA4 link tracker', function () {
       element.querySelector('.nothing').click()
       expect(window.dataLayer[2]).toEqual(undefined)
     })
+
+    it('tracks only links from multiple different CSS classes', function () {
+      element = document.createElement('div')
+      element.innerHTML =
+        '<a class="first" href="#link1">Link 1</a>' +
+        '<div class="trackme">' +
+          '<a class="second" href="#link2">Link 2</a>' +
+          '<a href="#link3"><span class="third">Link 3</span></a>' +
+          '<span class="nothing"></span>' +
+        '</div>' +
+        '<div class="trackmetwo">' +
+          '<a class="second" href="#link2">Link 2</a>' +
+          '<a href="#link3"><span class="third">Link 3</span></a>' +
+          '<span class="nothing"></span>' +
+        '</div>' +
+        '<div class="trackmethree">' +
+          '<a class="second" href="#link2">Link 2</a>' +
+          '<a href="#link3"><span class="third">Link 3</span></a>' +
+          '<span class="nothing"></span>' +
+        '</div>' +
+        '<div class="trackmefour">' +
+          '<a class="second" href="#link2">Link 2</a>' +
+          '<a href="#link3"><span class="third">Link 3</span></a>' +
+          '<span class="nothing"></span>' +
+        '</div>'
+
+      element.setAttribute('data-ga4-track-links-only', '')
+      element.setAttribute('data-ga4-limit-to-element-class', 'trackme, trackmetwo, trackmethree, trackmefour')
```

Comment:
@andysellick Thanks - I think it's because there'll be four classes used in `govspeak` that I went for that many. I'll meet you in the middle and get rid of one.